scipy/io/mmio.py

```diff
@@ -30,7 +33,7 @@ def mminfo(source):
     Parameters
     ----------
 
-    source : file
+    filename : file
```

Comment:
Don't forget that in python, changing the name of an argument is a backwards-incompatible API change.